Data/ByteString/Builder/Prim.hs

```diff
@@ -100,7 +100,7 @@ import           Data.ByteString.Builder.Prim
 
 renderString :: String -\> Builder
 renderString cs =
-    B.charUtf8 \'\"\' \<\> E.'encodeListWithB' escape cs \<\> B.charUtf8 \'\"\'
+    B.charUtf8 \'\"\' \<\> E.'primMapListBounded' escape cs \<\> B.charUtf8 \'\"\'
```

Comment:
I think this `E` should now be `P`.

```suggestion
    B.charUtf8 \'\"\' \<\> P.'primMapListBounded' escape cs \<\> B.charUtf8 \'\"\'
```

It also appears a few more times below.

Reply:
Good catch, there were actually a lot.

Reply:
Cheers! Could you possibly update the import statement above too, so the various identifiers would be in scope?

Please do say "Stop" when you've had enough BTW! :smile: There's a lot of stuff to fix here, and you don't have to do it all!

Reply:
Why not! I fixed the rest of the missing links (except internal ones frozen in comment blocks).